docker-compose.yml

```diff
@@ -5,3 +5,15 @@ services:
     container_name: beyond
     ports:
       - "5000:5000"
+
+  postgres:
```

Comment:
I would suggest we use a general service name, like: db.
Today we use postgress, tomorrow we might use a different vendor. We do not want to tie the service name with the vendor.

Reply:
> I would suggest we use a general service name, like: db.
> Today we use postgress, tomorrow we might use a different vendor. We do not want to tie the service name with the vendor.

Agree. changed the name here + in the app (where it is being referenced).